desktop/cmp/panel/impl/dragger/DraggerModel.js

```diff
@@ -24,6 +23,7 @@ export class DraggerModel {
     maxSize = null;
 
     constructor(panelModel) {
+        super();
```

Comment:
Does the use of `createObservableRef()` require a call to `makeObservable() here`, or is that all encapsulated within the object returned by this function? I.e. that object is itself observable, this class does not need to be...

I think that must be the case, but checking to be sure as the implementation of `createObservableRef` is a bit 🤯

Reply:
definitely not -- the reference to the object returned is not the thing that typically gets observed -- its typically static set once on any object.  

The thing that is observable is ObservableRef's  "current" property....